tools/ProdToConfig.pl

```diff
@@ -33,24 +34,26 @@
        $0 -help to list options
 USAGE
 &Getopt::Tabular::SetHelp( $Help, $Usage );
-&Getopt::Tabular::GetOptions( \@opt_table, \@ARGV ) || exit 1;
+&Getopt::Tabular::GetOptions( \@opt_table, \@ARGV )
+    || exit $NeuroDB::ExitCodes::GETOPT_FAILURE;
 
 ################################################################
 ################ Get config setting#############################
 ################################################################
+if (!$profile ) {
```

Comment:
same comment as above; looks like you added the check separately here...
but 3 lines later, you still have the condition as ` if ( $profile && !@Settings::db ) {` 
whereras I believe in other scripts, this was just `if ( !@Settings::db ) {`